packages/squiggle-lang/src/rescript/Distributions/DistributionTypes.res

```diff
@@ -35,6 +38,7 @@ module Error = {
     | OperationError(err) => Operation.Error.toString(err)
     | PointSetConversionError(err) => SampleSetDist.pointsetConversionErrorToString(err)
     | SparklineError(err) => PointSetTypes.sparklineErrorToString(err)
+    | RequestedStrategyInvalidError => `Requested strategy invalid`
```

Comment:
This should be more specific. Maybe pass a string with it, as it could be invalid for different reasons.